### DISCUSSION TITLE
Hydrogen + Vite + Remix installation issue

### DISCUSSION THREAD

**Discussion by vitalijalbu** [chakra-ui/chakra-ui#8690] [General]
### Description

Hi guys, trying to install chakra ui into a hydrogen + vite + remix storefront... but getting this error, anyone can help? thanks 👀 .

I copied exact code from the docs page.

![Screenshot 2024-07-30 alle 09 41 49](https://github.com/user-attachments/assets/3c812ca0-faa0-4337-b5d3-c0653e169c45)

### Link to Reproduction

https://v2.chakra-ui.com/getting-started/remix-guide

### Chakra UI Version

latest

### Operating System

- [x] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
That guide is v2. v2 is maintenance-only.

For Remix + Vite, follow https://chakra-ui.com/docs/get-started/frameworks/remix and wrap the app in `ChakraProvider`.
